README.md

```diff
@@ -80,6 +80,7 @@ These are the steps I go through mentally in the interviews, followed by actual
 It generally depends what you are and you will be working on. Also what your level is but these are some of the more frequent interview questions.
 
 * Design amazon's frequently viewed product page (eg. which shows the last 5 items you saw)
+* Design a movie-ticket booking facility(like BookMyShow) with facilities like adding screen info,check seat availability,hold seat for user while transaction takes place,dynamic pricing according to movie's popularity, manage peak usage when a blockbuster releases.
```

Comment:
If it's okay, can you fix the spacing after the commas or just format with Grammarly?